File: BEE 1020.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var num = parseInt(lines.shift());

var year = parseInt(num / 365);
var month = parseInt((num % 365) / 30);
var day = (num % 365) % 30;

console.log(year+" ano(s)");
console.log(month+" mes(es)");
console.log(day+" dia(s)");

File: BEE 1020.cpp
#include<iostream>

using namespace std;

int main(){
    int num;
    cin >> num;
    int year = num / 365;
    int month = (num % 365) / 30;
    int day = (num % 365) % 30;
    cout << year << " ano(s)" << endl;
    cout << month << " mes(es)" << endl;
    cout << day << " dia(s)" << endl;
    return 0;
}